web/styles/compose.css

```diff
@@ -535,11 +535,49 @@ textarea.new_message_textarea,
     }
 }
 
-input.recipient_box {
-    margin: 0;
-    padding: 0 6px;
-    height: auto;
+#compose_recipient_box {
+    /* Recipient box is the div container that has input and button */
     border-radius: 3px;
+    display: flex;
+    flex-grow: 1;
+    background: hsl(0deg 0% 100%);
+
+    &.focused {
+        outline: 0;
+        border: 1px solid hsl(0deg 0% 67%);
+        box-shadow: none;
+    }
+
+    #stream_message_recipient_topic,
+    #recipient_box_new_topic_button {
+        background: none;
+        border: none;
+        color: inherit;
+    }
+
+    /* Styles for input in the recipient_box */
+    #stream_message_recipient_topic {
+        flex-grow: 1;
+        box-shadow: none;
+    }
+
+    /* Styles for new topic button in the recipient_box */
+    #recipient_box_new_topic_button {
+        flex-shrink: 0;
+        /* The button looks better if it have smaller radius than the parent */
+        border-radius: 2px;
+        padding: 3px 10px;
+        margin: 1px;
+
+        &:hover {
+            background-color: hsl(231deg 100% 90% / 20%);
```

Comment:
Is this a new redesign color? It might belong in a variable, if so (a quick grep doesn't show it elsewhere in the codebase)

Reply:
It's just that the color was used in the demo. @terpimost, could you assist us here? Should we utilize a variable from the existing list? Or should we introduce a new variable? Here's a screenshot of the background we're discussing: 
![image](https://github.com/zulip/zulip/assets/53193850/819769db-137d-48be-a2c3-845ab385a522)